packages/opentelemetry-metrics/src/types.ts

```diff
@@ -69,3 +69,13 @@ export const DEFAULT_METRIC_OPTIONS = {
   labelKeys: [],
   valueType: ValueType.DOUBLE,
 };
+
+export class LabelSet implements LabelSet {
```

Comment:
Few suggestions:

1. Add JSDoc comment
2. Move `LabelSet`  to separate module (LabelSet.ts) (?)
3. `LabelSet implements LabelSet` looks typo or is this intentional stuff?